### ISSUE TITLE
App to do a proper recovery from 403 error on authentication

### ISSUE THREAD

**Issue by sallymoc** [qubic/wallet-app#67]
![image](https://github.com/user-attachments/assets/7a620304-2d36-4cf9-b038-6cdbd0d3f0ce)

Reported in wallet support channel in discord.
iOS user reported that when he open and it reads my face or enter the password, he receives the error on the screenshot. He tried to fully close and reopen few times and still get same error (5.08.2024).

New comment on 16.08
"Version 1.1.1 (38). If I turn my phone off and on it works once and then shows the error from that point on and I can’t access it unless I turn the phone on off and on again"

**Comment by sallymoc** [comment]
Feedback given to the user in discord but still waiting he contact me via DM

sally — 08/17/2024 7:32 PM
@BoredBobb22 we saw there are some authentication requests blocked by Cloudfare. Most likely are yours. Please contact me to validate and share a couple of ideas

**Comment by sallymoc** [comment]
There is no way to recover from this, because it's just cloud fare blocking the request. We can improve the message to the user, but we will handle in general for connection errors in a separate ticket
